virttest/libvirt_xml/devices/vsock.py

```diff
@@ -0,0 +1,30 @@
+"""
+vsock device support class(es)
+
+https://libvirt.org/formatdomain.html#vsock
+"""
+
+from virttest.libvirt_xml import accessors
+from virttest.libvirt_xml.devices import base
+
+
+class Vsock(base.UntypedDeviceBase):
+
+    __slots__ = ('model_type', 'cid', 'address', 'alias')
+
+    def __init__(self, virsh_instance=base.base.virsh):
+        accessors.XMLAttribute('model_type', self,
+                               parent_xpath='/',
+                               tag_name='vsock',
+                               attribute='model')
+        accessors.XMLElementDict('cid', self,
+                               parent_xpath='/',
+                               tag_name='cid')
+        accessors.XMLElementDict('address', self,
```

Comment:
For address part, you may refer to controller.py, and refer to currently existed Address class